src/coreclr/jit/fgprofilesynthesis.h

```diff
@@ -37,6 +37,8 @@ class ProfileSynthesis
         p.Run(option);
     }
 
+    static PhaseStatus AdjustThrowEdgeLikelihoods(Compiler* compiler);
```

Comment:
Not sure if synthesis is the right place for this code; my only motivation for placing this here is to keep the heuristic's likelihood values in one place.